### PR TITLE
Add basic d3d11 bindings.

### PR DESCRIPTION
**Pull request by jdm**
This compiles but is untested so far.

### CODE REVIEW

File: openxr/src/graphics/d3d.rs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    }
+}
+
+#[derive(Copy, Clone)]
+pub struct Requirements {
+    pub adapter_luid: LUID,
+    pub min_feature_level: D3D_FEATURE_LEVEL,
+}
+
+pub struct SessionCreateInfo {
```
Comment (Ralith) [inline openxr/src/graphics/d3d.rs]: Any reason this shouldn't be `Copy + Clone`?

Comment (jdm) [reply openxr/src/graphics/d3d.rs]: Not that I can see.

Comment (Ralith) [reply openxr/src/graphics/d3d.rs]: Reasonable thing to expose by default, I think.

Comment (jdm) [reply openxr/src/graphics/d3d.rs]: Fixed.

**Pull request merged**